fix: add missing custom fields to order AddressTransformer

leptoquark1 opened the pull request:
<!--
Thank you for contributing to Shopware! Please fill out this description template to help us to process your pull request.

Please make sure to fulfil our contribution guideline (https://docs.shopware.com/en/shopware-platform-dev-en/community/contribution-guideline?category=shopware-platform-dev-en/community).

Do your changes need to be mentioned in the documentation?
Add notes on your change right now in the documentation files in /src/Docs/Resources and add them to the pull request as well. 
-->

### 1. Why is this change necessary?
Since the order address is a snapshot of the customer address, all custom fields that may be added to the respective customer address, should be cloned too.
With this change it's also possible to access the custom fields of the customer address when querying the order or one of its addresses via api.

### 2. What does this change do, exactly?
Add the customFields attribute of the customer address to the list of attributes to clone to create a order address.

### 3. Describe each step to reproduce the issue or behaviour.
-

### 4. Please link to the relevant issues (if any).
-

### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

mitelg commented:
Hi @mkreusch 
Thanks for your contribution!

Please have a look at our [Coding Standards](https://developers.shopware.com/developers-guide/coding-standards/#php-coding-standard) and reformat you code accordingly.
To fix the coding style you can use tools like [php-cs-fixer](http://cs.sensiolabs.org/) or [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer). Also PhpStorm can be configured to use PSR-1 and PSR-2 to reformat your code automatically.

Best regards!

mkreusch commented:
Hi @mitelg 
Sorry for that - i have fixed it.

OliverSkroblin commented:
Hello, thank you for your pull request, we have looked at again in more detail. We currently do not see the functionality inside the core and assume that this can be solved individually in the respective projects. Registering Smarty functions via theme or plugin is indeed easily possible. Looking ahead we will tackle the whole issue "static pages" again. Thank you very much for your understanding

Commit:
fix: add missing custom fields to order AddressTransformer

Since the order address is a snapshot of the customer address, all custom fields that may be added to the respective customer address, should be cloned too.
With this change it's also possible to access the custom fields of the customer address when querying the order or one of its addresses via api.

J-Rahe commented:
Hi @leptoquark1 

thank you for your contribution!
Your PR was merged to 6.2
